duradmin/src/main/webapp/WEB-INF/jsp/spaces/spaces-manager.jsp

```diff
@@ -294,13 +294,13 @@
               <a class=" download-manifest-button button" data-dropdown="#manifest-dropdown">
                 <i class="pre download" title="download manifest"></i>Manifest
               </a>
- 
+
```

Comment:
This formatting change probably should be reverted or at least added as a separate formatting themed commit since it is not related to duracloud-1214 (as far as I can tell).